examples/esplora_backend_synchronous.rs

```diff
@@ -0,0 +1,88 @@
+use std::str::FromStr;
+
+use bdk::blockchain::Blockchain;
+use bdk::{
+    blockchain::esplora::EsploraBlockchain,
+    database::MemoryDatabase,
+    template::Bip84,
+    wallet::{export::FullyNodedExport, AddressIndex},
+    KeychainKind, SyncOptions, Wallet,
+};
+use bitcoin::{
+    util::bip32::{self, ExtendedPrivKey},
+    Network,
+};
+
+pub mod utils;
+
+use crate::utils::tx::build_signed_tx;
+
+/// This will create a wallet from an xpriv and get the balance by connecting to an Esplora server.
```

Comment:
Similar suggestion. The idea is to explain what this example is for in the first line. 
```suggestion
/// This will create a wallet from an xpriv and get the balance by connecting to an Esplora server, using blocking calls with `ureq`.  
```

Reply:
Done in e50fcb6c53964435d8a70f6fe396bf60bb75f1e2.